aws/src/main/java/org/apache/iceberg/aws/s3/signer/S3V4RestSignerClient.java

```diff
@@ -151,34 +165,37 @@ private RESTClient httpClient() {
     return httpClient;
   }
 
-  @VisibleForTesting
-  AuthSession authSession() {
-    if (null == authSession) {
+  private AuthManager authManager() {
```

Comment:
could you please move the `authManager()` method just below `authSession()` as it's causing a diff that adds visual overhead during review